GitVersionCore.Tests/IntegrationTests/GitFlow/DevelopScenarios.cs

```diff
@@ -6,29 +6,43 @@
 public class DevelopScenarios
 {
     [Test]
-    public void WhenDevelopBranchedFromMaster_MinorIsIncreased()
+    public void WhenDevelopBranchedFromTaggedCommitOnMasterVersionDoesNotChange()
     {
         using (var fixture = new EmptyRepositoryFixture(new Config()))
         {
             fixture.Repository.MakeATaggedCommit("1.0.0");
             fixture.Repository.CreateBranch("develop").Checkout();
+            // TODO Should actually be 1.0.0+0
```

Comment:
I think we have a bug at the moment.

If I tag master, then branch develop from that tag it gets an unstable pre-release tag. It shouldn't because that commit is tagged.